src/middleware/validateSchema.js

```diff
@@ -3,16 +3,15 @@
 const {
 	isBoolean, includes, map, has, get
 } = require('lodash')
-const { schemas } = require('../lib')
+const  schemas  = require('../schema')
 
 module.exports = (shouldUseJoiError = false) => {
```

Comment:
so to move away from the route-based approach. we will be passing the specific schema from the route

so kindly introduce a second parameter called `schema`. so you'll be taking anything related to the route stuff. will flag them